core/modules/layout/plugins/access/layout_access.inc

```diff
@@ -153,11 +153,17 @@ class LayoutAccessNegatable extends LayoutAccess {
    */
   function form(&$form, &$form_state) {
     parent::form($form, $form_state);
+    $options = array(
+      TRUE => t('Is NOT front page'),
+      FALSE => t('Is front page'),
+    );
     $form['negate'] = array(
```

Comment:
I believe there's no point in still calling this form "negate" after the change we are making. It would be confusing to keep it like so.